packages/ovh-manager-core/translate/translate.service.js

```diff
@@ -0,0 +1,116 @@
+import _ from 'lodash';
```

Comment:
You should use destructuration

Example:
```
import { get } from 'lodash/get'
```

Importing lodash totally have a big impact on bundle size :/